crun.spec.rpkg

```diff
@@ -0,0 +1,86 @@
+# For automatic rebuilds in COPR
+
+# The following tag is to get correct syntax highlighting for this file in vim text editor
+# vim: syntax=spec
+
+# git_dir_name returns repository name derived from remote Git repository URL
+Name:       {{{ git_dir_name }}}
+
+Epoch: 101
+
+# git_dir_version returns version based on commit and tag history of the Git project
+Version:    {{{ git_dir_version }}}
+
+# This can be useful later for adding downstream patches
+Release:    1%{?dist}
+
+# Basic description of the package
+Summary: Manage Pods, Containers and Container Images
+
+# License. We assume GPLv2+ here.
+License:   GPLv2+
+
+# Home page of the project. Can also point to the public Git repository page.
+URL:        https://github.com/containers/%{name}
+
+# Detailed information about the source Git repository and the source commit
+# for the created rpm package
+VCS:        {{{ git_dir_vcs }}}
+
+# git_dir_pack macro places the repository content (the source files) into a tarball
+# and returns its filename. The tarball will be used to build the rpm.
+Source:     {{{ git_dir_pack }}}
+
+# We always run autogen.sh
+BuildRequires: autoconf
+BuildRequires: automake
+BuildRequires: gcc
+BuildRequires: python3
+BuildRequires: libcap-devel
+BuildRequires: systemd-devel
+BuildRequires: yajl-devel
+BuildRequires: libseccomp-devel
+BuildRequires: libselinux-devel
+BuildRequires: python3-libmount
+BuildRequires: libtool
+BuildRequires: go-md2man
+BuildRequires: make
+BuildRequires: glibc-static
+BuildRequires: protobuf-c-devel
+%ifnarch %ix86
+BuildRequires: criu-devel >= 3.15
+%endif
+Provides: oci-runtime
+
+# More detailed description of the package
+%description
+%{name} is a runtime for running OCI containers
+
+# The following four sections already describe the rpm build process itself.
+# prep will extract the tarball defined as Source above and descend into it.
+%prep
+{{{ git_dir_setup_macro }}}
+
+# This will invoke `make` command in the directory with the extracted sources.
+%build
+./autogen.sh
+%configure --disable-silent-rules
```

Comment:
@giuseppe is there a convenient makefile target i can use here to handle building everything under libocispec here? Currently `rpkg local` doesn't work and complains about header files not found in libocispec.

Reply:
Is it possible to fetch remote git repositories?  libocispec is a git submodule so it is empty until you run `git submodule update`. Alternatively could we use `make dist`?

Reply:
reusing existing spec file in latest change.